domains/lexio.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "lexiolvsu",
+           "email": "",
+           "discord": "598529040172843019",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.n6Dm3GKmIx4VcrtqItR5n7JsMo8dKGxYzzUs4Z3klgMqcs0xD6JMpLE-HRh7FRBm4UmRMwv_tBjWxTLmsgjoYQOihU9fybnOKN_3qJqrcGsriG-p8FHALZN39rT-X5MCSIiFFuv3-2L8R8qikWun_J18sifcJASq8MW5ZZxzzd0P7CdRrJ42Z4zO88Bep9Yg468hHZRCFe5eqmpGxN2nKuI5qptlC7W01-svwIXUeLFhrDuFSy-2x0Jt4ttqyOqm2lYlyaqOmoZO-4Dvwdh30xcwJdHOZnSi3FW7v7l0VEvsju34jJCa6uCzvJaDUbHpwdfQdPPgNTsXNAwf0UJogw.hWggfWaA4h3VqDWEyu1sJA.jTlfwj4JYPZNN20R1ckxJCxap1JfxvE4fNwX5zCFBIt32NwFKTEzhUIM3Cjgrn4cGex22n6uEJautBYEnBbYnt0Tecp6PnJrNq-iIZuU0b8.LfoC83nTM5xhCE43OUsGKQ"
+        },
+    
+        "record": {
+            "CNAME": "lexio.pages.dev"
```

Comment:
Uh, I would like you to add more content to the page. Since site is basically a WIP I'd recommend you try to register the domain after its been done.